libraries/rush-lib/assets/rush-init-subspace/.pnpmfile.cjs

```diff
@@ -0,0 +1,38 @@
+'use strict';
```

Comment:
Is this the same as the one in `rush init`. Worth duplicating?

Reply:
Changed to use the same files as `rush init`